nixos/modules/services/misc/mediatomb.nix

```diff
@@ -164,15 +168,24 @@ in {
 
       serverName = mkOption {
         type = types.str;
-        default = "mediatomb";
+        default = "Gerbera (Mediatomb)";
         description = ''
           How to identify the server on the network.
         '';
       };
 
+      package = mkOption {
+        type = types.package;
+        example = literalExample "pkgs.mediatomb";
```

Comment:
Given the wide range of options available it might be more valuable to show an example with an `override`.